.github/workflows/cd.yml

```diff
@@ -0,0 +1,49 @@
+# This CD workflow is adapted from:
+# scientific-python/cookie (2024-06-28), BSD-3-Clause license
+# Refs:
+# - https://github.com/scientific-python/cookie/blob/23926fe2b88af44377f20afb3b11d63afb7fe315/.github/workflows/cd.yml
+# - https://learn.scientific-python.org/development/guides/gha-pure/
+
+name: CD
+
+on:
+  push:
+    tags:
+      - v*.*.*
+
+jobs:
+  dist:
+    # Build the Python SDist and wheel, performs metadata and readme linting
+    # Ref: https://github.com/hynek/build-and-inspect-python-package
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: hynek/build-and-inspect-python-package@v2
```

Comment:
Do you want to generate attestations?
Ref: https://github.com/hynek/build-and-inspect-python-package?tab=readme-ov-file#inputs